src/hw5/num1/Student.java

```diff
@@ -0,0 +1,45 @@
+package hw5.num1;
+
+public class Student {
+
+    private String name;
+    private int course;
+    private int grade;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
```

Comment:
public void setName(String name) {
    if (name.length() >= 3) {
      this.name = name;
    }